Improved design for desktop and mobile view and added icons

MuditJain5 opened the pull request:
Improved design for desktop and mobile view and added icons wherever needed

Sidhegde19 commented:
My friend, the feature of icons has already been addressed. If you can make some other improvements, please do address them. Sorry for the inconvenience

Sidhegde19 commented:
For the mobile view, please submit another PR so as reduce conflicts. Thank you

MuditJain5 commented:
Sorry for not mentioning the changes.
Changes :
1. Added padding and spaces
2. Corrected both the image size 
3. Correctly aligned the footer content
4. Corrected the location of map div
5. Improved button design 
6. Changed font sizes and weight
